components/front-page/News.tsx

```diff
@@ -0,0 +1,53 @@
+import { News } from "lib/cms/get-news";
+import { isCurrentOrigin } from "lib/util/is-current-origin";
+import Image from "next/image";
+import Link from "next/link";
+
+export const NewsSection = ({
+  news,
+  imagePlaceholders,
+}: {
+  news: News[];
+  imagePlaceholders: string[];
+}) => {
+  return (
+    <div className="mb-12">
```

Comment:
Hover effect slightly cuts off card near the mid-range breakpoint (below 905px)

![Screenshot 2023-09-05 at 9 40 50 PM](https://github.com/zeitgeistpm/ui/assets/14626415/58ba3269-7850-4993-acef-06129c1a7165)

Reply:
Yeah was the same for market cards. I just disabled the hover effects for mobile since its not really necessary when its touch event based, actually kinda looks weird when it gets enlarged when clicked/touched as opposed to hovered.

Reply:
Good call!